apps/web/src/pages/Home/IAPHomePage/Home.tsx

```diff
@@ -490,17 +535,28 @@ export const Home = ({ latestPool }: HomeProps) => {
           backgroundSize: "cover",
           backgroundPosition: "top center",
         }}
+        role="img"
+        aria-label={intl.formatMessage({
+          defaultMessage: "Traditional Métis sash",
+          id: "g6Atld",
+          description: "Indigenous Apprenticeship sash image text alternative",
+        })}
       >
         <div data-h2-container="base(center, iap-home, x1) p-tablet(center, iap-home, x2)">
           <div>
             <div data-h2-shadow="base(l)" data-h2-flex-grid="base(stretch, 0)">
               <div data-h2-flex-item="base(1of1) p-tablet(1of3) l-tablet(1of2) desktop(4of7)">
-                <div
+                <img
```

Comment:
Hmm, why did this one get taller?

Before:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/b2e7463e-a331-4325-9f56-c716a7f03ce4)

After:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/0ca41a8f-d79d-432c-82ed-7ab06f8944d4)